ci_support/run_docker_build.sh

```diff
@@ -44,6 +44,9 @@ conda install --yes conda-build
 conda info
 
 # Embarking on 1 case(s).
+    set -x
+    export CONDA_PERL=5.20.3.1
+    set +x
```

Comment:
This comes out of these [lines]( https://github.com/conda-forge/conda-smithy/blob/211505affe36d8b420ce5ece1860ec401e841daf/conda_smithy/templates/run_docker_build_matrix.tmpl#L12-L14 ) in `conda-smithy`. Given it is just setting environment variable maybe these are not needed.